QUEST_LV_0100.tsv

```diff
@@ -13113,7 +13113,7 @@ QUEST_LV_0100_20151224_013109	Melchioras has managed to regain his conscience.{n
 QUEST_LV_0100_20151224_013110	Melchioras regained his conscience while we were fighting, it seems.{nl}I'm glad everything is going to be alright now. I really am.
 QUEST_LV_0100_20151224_013111	Melchioras was asking for you the whole time.{nl}I don't even know what to say, I'm so glad everything ended well...
 QUEST_LV_0100_20151224_013112	I hoped I could be the one to catch Delmore Rephaim...{nl}But I'm glad it turned out well. If only I wasn't hurt...
-QUEST_LV_0100_20151224_013113	Chasing away the demons, rebuilding the walls...{nl}I wonder how much time it would take to make Delmore Fortress a place worth living again.
+QUEST_LV_0100_20151224_013113	Chasing away the demons, rebuilding the walls...{nl}I wonder how much time it would take to make the Delmore Castle a place worth living again.
```

Comment:
I wonder how long will it take to make the Delmore Castle a place worth living again.